app/views/comments/index.html.erb

```diff
@@ -37,96 +37,113 @@
   <% end %>
 <% end %>
 
-<% if @commentable.class.name == "Article" %>
-  <%= render "shared/payment_pointer", user: @commentable.user %>
-  <div id="comment-article-indicator" data-article-id="<%= @commentable.id %>">
-<% end %>
-<% if @root_comment %>
-  <div class="single-comment-header"></div>
-<% else %>
-  <div class="article-header">
-    <a href="<%= @commentable.path %>" class="header-link">
-      <% if @commentable.respond_to?(:main_image) && @commentable.main_image.present? %>
-        <div class="picture" style="background-image:url(<%= cloud_cover_url(@commentable.main_image) %>)"></div>
-      <% else %>
-        <div class="blank-comment-space"></div>
-      <% end %>
-      <h3 id="comments-header"><%= @commentable.title %></h3>
-      <h4>
-        <%= @commentable.is_a?(PodcastEpisode) ? @commentable.podcast.name : @commentable.user.name %>
-        <span class="published-at"><%= "on #{@commentable.published_at.strftime('%B %d, %Y')}" if @commentable.published_at %></span>
-      </h4>
-    </a>
-    <% if @commentable.processed_html.present? %>
-      <% if @commentable.processed_html.size < 350 %>
-        <div class="body">
-          <%= sanitize_rendered_markdown(@commentable.processed_html) %>
+<div class="crayons-layout crayons-layout--limited-l gap-0">
+  <% if @commentable.class.name == "Article" %>
```

Comment:
I wonder if this `if` can move into the payment pointer partial  so the logic on whether or not it renders something is self-contained. Then instead of passing in the user, we'd pass in the whole `commentable`. But I don't know all use cases of this, so maybe that doesn't make sense.

Reply:
Yea, since this partial is being used in other places too, I think I'll just pass on it for now. Someone more experienced than myself should chime-in (in a separate PR) and optimize it if it makes sense :)